src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public abstract class Machine {
+
```

Comment:
[Please don't add redundant empty lines to your code.](https://github.com/mate-academy/jv-oop/blob/master/checklist.md#please-dont-add-redundant-empty-lines-to-your-code)